app/src/main/res/values/strings.xml

```diff
@@ -10,7 +10,14 @@
     <string name="error_invalid_password">This password is too short</string>
     <string name="error_incorrect_password">This password is incorrect</string>

```

Comment:
remove the incorrect password, and add a new string that indicates incorrect credentials